examples/data-objects/inventory-app/src/inventoryList.ts

```diff
@@ -6,24 +6,24 @@
 import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
 import {
 	ForestType,
-	ISharedTree,
-	SharedTreeFactory,
-	TreeView,
+	TreeFactory,
+	TreeViewOld,
 	typeboxValidator,
+	ITree,
 } from "@fluid-experimental/tree2";
 import { IFluidHandle } from "@fluidframework/core-interfaces";
 import { Inventory, treeConfiguration } from "./schema";
 
 const treeKey = "tree";
 
-const factory = new SharedTreeFactory({
+const factory = new TreeFactory({
```

Comment:
Nit: how tractable would it be to split some of the renames out into separate PR(s)? Might help narrow the scope of the changes here a bit.

Reply:
This is not a rename. This is using a new type that needed a different name to not collide. There still is a SharedTreeFactory, and it still works just fine.

The inventory list sample could be left using the old non-class based API in this PR: it still works just fine, but I figured porting it would help prove things worked.

Reply:
The only things that were renamed in this PR were APIs for the old setup that collided with one for the new one, as doing so allowed updating a few places to avoid having to do something like "schematize2" then rename it back later.

There were only a few cases where this happened, so it seemed reasonable and allowed the new API to showcase its final form.